numpy/distutils/ccompiler.py

```diff
@@ -620,7 +655,26 @@ def matcher(version_string):
             version = m.group('version')
             return version
 
-    status, output = exec_command(version_cmd, use_tee=0)
+    try:
+        if not is_sequence(version_cmd):
+            version_cmd = version_cmd.split(' ')
+        output = subprocess.check_output(version_cmd)
+    except subprocess.CalledProcessError as exc:
+        output = exc.output
+        status = exc.returncode
+    except (OSError, AttributeError):
```

Comment:
`AttributeError` can probably be extracted now that `is_sequence` guards `version_cmd` here too